src/types-and-traits.rst

```diff
@@ -2637,6 +2637,28 @@ A :t:`higher-ranked trait bound` is a :t:`bound` that specifies an infinite
 list of :t:`[bound]s` for all possible :t:`[lifetime]s` specified by the
 :s:`ForGenericParameterList`.
 
+:dp:`fls_J9DEsd06Ttu9`
+An :t:`implied bound` is a :t:`bound` that is not expressed in syntax, but is
+is the byproduct of relations between the :t:`[generic parameter]s` of
+:t:`[abstract data type]s` or :t:`[function]s`.
+
+:dp:`fls_IfHRxSasGAih`
+A :t:`reference` of the form ``&'a generic_parameter``, where ``'a`` is a
+:t:`lifetime parameter` and ``generic_parameter`` is an arbitrary
+:t:`generic parameter`, yields :t:`implied bound``
+``generic_parameter: 'a``.
+
+:dp:`fls_K8nPGP5xbLb7`
+If an :t:`implied bound` applies to the :t:`type` of a :t:`field`, then the
+:t:`implied bound` also applies to the :t:`abstract data type` where the
+:t:`field` resides.
```

Comment:
The type of a field can also have an explicit lifetime bound which in turn will become an implied lifetime bound for the ADT here, that is outlives requirement on  the field type, be it explicit or implied are added as an implied bound to the containing ADT

Reply:
Fixed.